client/www/components/dash/explorer/EditRowDialog.tsx

```diff
@@ -788,6 +788,15 @@ export function EditRowDialog({
     });
   };
 
+  // Auto-resize textareas when dialog opens
+  useLayoutEffect(() => {
```

Comment:
Maybe we extract this code out into a `ResizingTextArea`? or something like that? 

(For this kind of extending text area usually the way to go use to use contenteditable inputs, etc. But heck if this works well let's just use it!)